src/ci/docker/host-x86_64/dist-armv7-linux/Dockerfile

```diff
@@ -19,9 +19,6 @@ USER root
 COPY scripts/sccache.sh /scripts/
 RUN sh /scripts/sccache.sh
 
-COPY scripts/cmake.sh /scripts/
-RUN /scripts/cmake.sh
```

Comment:
Curious: what changed to remove the necessity of building our own cmake?

Reply:
#93837 moved it to build wholly on `ubuntu:20.04`, which has a new enough cmake already. We could have removed this custom cmake at that time, but didn't notice. I saw it now as I was looking over all the crosstool-ng builders.

Reply:
Ok great!